rare/components/tabs/games/env_vars.py

```diff
@@ -0,0 +1,97 @@
+from PyQt5.QtWidgets import QGroupBox, QTableWidgetItem
+from rare.ui.components.tabs.games.env_vars import Ui_env_vars_groupbox
+from rare.utils import config_helper
+from rare.shared import LegendaryCoreSingleton
+
+class EnvVars(QGroupBox, Ui_env_vars_groupbox):
+    def __init__(self, parent):
+        super(EnvVars, self).__init__(parent=parent)
+        self.setupUi(self)
+        self.env_vars_table.cellChanged.connect(self.create_new_row)
+        self.env_vars_table.cellChanged.connect(self.update_env_vars)
+        self.env_vars_table.verticalHeader().sectionClicked.connect(self.remove_row)
+        self.app_name = None
+        self.list_of_labels = []
+    
+    def import_env_vars(self):
+        self.env_vars_table.clearContents()
+
+        list_of_keys = []
+        list_of_values = []
+ 
+        # First, we try to get all keys and values from the config
+        try:
+            for key in LegendaryCoreSingleton().lgd.config[f"{self.app_name}.env"]:
```

Comment:
Maybe add a self.core attribute instead of use everytime the Singleton Object

Reply:
Done.